mrmustard/math/backend_numpy.py

```diff
@@ -14,7 +14,7 @@
 
 """This module contains the numpy backend."""
 
-# pylint: disable = missing-function-docstring, missing-class-docstring, fixme
+# pylint: disable = missing-function-docstring, missing-class-docstring, fixme, too-many-positional-arguments
```

Comment:
Btw looking at the [pylint documentation](https://pylint.readthedocs.io/en/latest/user_guide/messages/refactor/too-many-positional-arguments.html) the recommendation is to change the signature to 
```
    def convolution(
        self,
        *,
        array: np.ndarray,
        ...
```
Personally I don't have a strong opinion either way so feel free to resolve this comment